contrib/masmx86/bld_ml32.bat

```diff
@@ -1,2 +1,2 @@
-ml /coff /Zi /c /Flmatch686.lst match686.asm
-ml /coff /Zi /c /Flinffas32.lst inffas32.asm
+ml /coff /Zi /c /safeseh /Flmatch686.lst match686.asm
+ml /coff /Zi /c /safeseh /Flinffas32.lst inffas32.asm

```

Comment:
What is the reason for adding /safeseh for older versions of Visual Studio... Is it actually supported and required in all versions?